include/head/core/communication.hpp

```diff
@@ -0,0 +1,132 @@
+#pragma once
+
+#include <array>
+#include <span>
+
+#include "common/core/bit_utils.hpp"
+#include "common/core/io.hpp"
```

Comment:
This is leftover from a uart task which we don't have anymore. We should delete this file.